Error cross building runc for riscv64 arch

First encountered on `v0.10` branch: https://github.com/moby/buildkit/pull/3587#pullrequestreview-1285116734

```
#16 [linux/amd64 internal] load metadata for docker.io/library/alpine:3.15
------
 > [linux/amd64->riscv64 runc 3/3] RUN --mount=from=runc-src,src=/usr/src/runc,target=. --mount=target=/root/.cache,type=cache   CGO_ENABLED=1 xx-go build -mod=vendor -ldflags '-extldflags -static' -tags 'apparmor seccomp netgo cgo static_build osusergo' -o /usr/bin/runc ./ &&   xx-verify --static /usr/bin/runc:
#61 133.0 /usr/bin/riscv64-alpine-linux-musl-ld: failed to merge target specific data of file $WORK/b192/_x006.o
#61 133.0 /usr/bin/riscv64-alpine-linux-musl-ld: failed to merge target specific data of file $WORK/b192/_x007.o
#61 133.0 /usr/bin/riscv64-alpine-linux-musl-ld: failed to merge target specific data of file $WORK/b192/_x008.o
#61 133.0 /usr/bin/riscv64-alpine-linux-musl-ld: failed to merge target specific data of file $WORK/b192/_x009.o
#61 133.0 /usr/bin/riscv64-alpine-linux-musl-ld: failed to merge target specific data of file $WORK/b192/_x010.o
#61 133.0 /usr/bin/riscv64-alpine-linux-musl-ld: -march=rv64i2p1_m2p0_a2p1_f2p2_d2p2_c2p0_zicsr2p0_zifencei2p0_zmmul1p0: unknown z ISA extension `zmmul'
#61 133.0 /usr/bin/riscv64-alpine-linux-musl-ld: failed to merge target specific data of file /riscv64-alpine-linux-musl//usr/lib/gcc/riscv64-alpine-linux-musl/12.2.1/crtendS.o
#61 133.0 /usr/bin/riscv64-alpine-linux-musl-ld: -march=rv64i2p1_m2p0_a2p1_f2p2_d2p2_c2p0_zicsr2p0_zifencei2p0_zmmul1p0: unknown z ISA extension `zmmul'
#61 133.0 /usr/bin/riscv64-alpine-linux-musl-ld: failed to merge target specific data of file /riscv64-alpine-linux-musl//usr/lib/gcc/riscv64-alpine-linux-musl/12.2.1/crtn.o
#61 133.0 clang-12: error: linker command failed with exit code 1 (use -v to see invocation)
```

But we now have the same issue on master branch: https://github.com/moby/buildkit/actions/runs/4152157541/jobs/7185004841#step:6:520

It might be related to `libseccomp` or `musl` packages https://github.com/moby/buildkit/blob/31dde702007034f376eeaada350fe0a2bb45a439/Dockerfile#L62

that are installed from Alpine edge repo with `xx`: https://github.com/tonistiigi/xx/blob/82ed2822306002b6b65b6ff9eaab2f30268c5a6d/base/xx-apk#L49-L53

It works in https://github.com/moby/moby/pull/44735 when building on `ubuntu:22.04`:
* https://github.com/moby/moby/actions/runs/4154514752/jobs/7186964870#step:5:3278
* https://github.com/moby/moby/actions/runs/4154514752/jobs/7186964870#step:5:3675

I tried to just use the [`make static` target](https://github.com/opencontainers/runc/blob/5fd4c4d144137e991c4acebb2146ab1483a97925/Makefile#L40-L41) in our Dockerfile but same issue.

Also tried to build https://github.com/seccomp/libseccomp and similar issue:

```
#20 [libseccomp 7/7] RUN --mount=from=libseccomp-src,src=/usr/src/libseccomp,target=.,rw <<EOT (set -ex...)
#0 0.139 + xx-clang --print-target-triple
#20 0.144 + DESTDIR=/out CC=xx-clang ./configure '--host=riscv64-alpine-linux-musl' --enable-static --enable-shared
#20 0.170 checking for a BSD-compatible install... /usr/bin/install -c
#20 0.303 checking whether build environment is sane... yes
#20 0.307 checking for riscv64-alpine-linux-musl-strip... no
#20 0.307 checking for strip... strip
#20 0.307 checking for a thread-safe mkdir -p... build-aux/install-sh -c -d
#20 0.311 checking for gawk... no
#20 0.311 checking for mawk... no
#20 0.311 checking for nawk... no
#20 0.311 checking for awk... awk
#20 0.311 checking whether make sets $(MAKE)... yes
#20 0.315 checking whether make supports nested variables... yes
#20 0.317 checking how to create a pax tar archive... gnutar
#20 0.322 checking for riscv64-alpine-linux-musl-gcc... xx-clang
#20 0.411 checking whether the C compiler works... no
#20 0.456 configure: error: in `/usr/src/libseccomp':
#20 0.456 configure: error: C compiler cannot create executables
#20 0.456 See `config.log' for more details
#20 0.464 + cat config.log
#20 0.464 This file contains any messages produced by compilers while
#20 0.464 running configure, to aid debugging if configure makes a mistake.
#20 0.464
#20 0.464 It was created by libseccomp configure 2.5.4, which was
#20 0.464 generated by GNU Autoconf 2.69.  Invocation command line was
#20 0.464
#20 0.464   $ ./configure --host=riscv64-alpine-linux-musl --enable-static --enable-shared
#20 0.464
#20 0.464 ## --------- ##
#20 0.464 ## Platform. ##
#20 0.464 ## --------- ##
#20 0.464
#20 0.464 hostname = buildkitsandbox
#20 0.464 uname -m = x86_64
#20 0.464 uname -r = 5.15.79.1-microsoft-standard-WSL2
#20 0.464 uname -s = Linux
#20 0.464 uname -v = #1 SMP Wed Nov 23 01:01:46 UTC 2022
#20 0.464
#20 0.464 /usr/bin/uname -p = unknown
#20 0.464 /bin/uname -X     = unknown
#20 0.464
#20 0.464 /bin/arch              = x86_64
#20 0.464 /usr/bin/arch -k       = unknown
#20 0.464 /usr/convex/getsysinfo = unknown
#20 0.464 /usr/bin/hostinfo      = unknown
#20 0.464 /bin/machine           = unknown
#20 0.464 /usr/bin/oslevel       = unknown
#20 0.464 /bin/universe          = unknown
#20 0.464
#20 0.464 PATH: /usr/local/sbin
#20 0.464 PATH: /usr/local/bin
#20 0.464 PATH: /usr/sbin
#20 0.464 PATH: /usr/bin
#20 0.464 PATH: /sbin
#20 0.464 PATH: /bin
#20 0.464
#20 0.464
#20 0.464 ## ----------- ##
#20 0.464 ## Core tests. ##
#20 0.464 ## ----------- ##
#20 0.464
#20 0.464 configure:2225: checking for a BSD-compatible install
#20 0.464 configure:2293: result: /usr/bin/install -c
#20 0.464 configure:2304: checking whether build environment is sane
#20 0.464 configure:2359: result: yes
#20 0.464 configure:2418: checking for riscv64-alpine-linux-musl-strip
#20 0.464 configure:2448: result: no
#20 0.464 configure:2458: checking for strip
#20 0.464 configure:2474: found /usr/bin/strip
#20 0.464 configure:2485: result: strip
#20 0.464 configure:2510: checking for a thread-safe mkdir -p
#20 0.464 configure:2549: result: build-aux/install-sh -c -d
#20 0.464 configure:2556: checking for gawk
#20 0.464 configure:2586: result: no
#20 0.464 configure:2556: checking for mawk
#20 0.464 configure:2586: result: no
#20 0.464 configure:2556: checking for nawk
#20 0.464 configure:2586: result: no
#20 0.464 configure:2556: checking for awk
#20 0.464 configure:2572: found /usr/bin/awk
#20 0.464 configure:2583: result: awk
#20 0.464 configure:2594: checking whether make sets $(MAKE)
#20 0.464 configure:2616: result: yes
#20 0.464 configure:2645: checking whether make supports nested variables
#20 0.464 configure:2662: result: yes
#20 0.464 configure:2741: checking how to create a pax tar archive
#20 0.464 configure:2752: tar --version
#20 0.464 tar (GNU tar) 1.34
#20 0.464 Copyright (C) 2021 Free Software Foundation, Inc.
#20 0.464 License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
#20 0.464 This is free software: you are free to change and redistribute it.
#20 0.464 There is NO WARRANTY, to the extent permitted by law.
#20 0.464
#20 0.464 Written by John Gilmore and Jay Fenlason.
#20 0.464 configure:2755: $? = 0
#20 0.464 configure:2795: tardir=conftest.dir && eval tar --format=posix -chf - "$tardir" >conftest.tar
#20 0.464 configure:2798: $? = 0
#20 0.464 configure:2802: tar -xf - <conftest.tar
#20 0.464 configure:2805: $? = 0
#20 0.464 configure:2807: cat conftest.dir/file
#20 0.464 GrepMe
#20 0.464 configure:2810: $? = 0
#20 0.464 configure:2823: result: gnutar
#20 0.464 configure:2882: checking for riscv64-alpine-linux-musl-gcc
#20 0.464 configure:2909: result: xx-clang
#20 0.464 configure:3178: checking for C compiler version
#20 0.464 configure:3187: xx-clang --version >&5
#20 0.464 Alpine clang version 15.0.7
#20 0.464 Target: riscv64-alpine-linux-musl
#20 0.464 Thread model: posix
#20 0.464 InstalledDir: /usr/bin
#20 0.464 Configuration file: /usr/bin/riscv64-alpine-linux-musl.cfg
#20 0.464 configure:3198: $? = 0
#20 0.464 configure:3187: xx-clang -v >&5
#20 0.464 Alpine clang version 15.0.7
#20 0.464 Target: riscv64-alpine-linux-musl
#20 0.464 Thread model: posix
#20 0.464 InstalledDir: /usr/bin
#20 0.464 Configuration file: /usr/bin/riscv64-alpine-linux-musl.cfg
#20 0.464 Found candidate GCC installation: /riscv64-alpine-linux-musl/usr/lib/gcc/riscv64-alpine-linux-musl/12.2.1
#20 0.464 Selected GCC installation: /riscv64-alpine-linux-musl/usr/lib/gcc/riscv64-alpine-linux-musl/12.2.1
#20 0.464  "/usr/bin/riscv64-alpine-linux-musl-ld" --sysroot=/riscv64-alpine-linux-musl/ -pie -z now -z relro --hash-style=gnu --build-id --eh-frame-hdr -m elf64lriscv -X -dynamic-linker /lib/ld-musl-riscv64.so.1 -o a.out /riscv64-alpine-linux-musl/usr/lib/Scrt1.o /riscv64-alpine-linux-musl/usr/lib/gcc/riscv64-alpine-linux-musl/12.2.1/crti.o /riscv64-alpine-linux-musl/usr/lib/gcc/riscv64-alpine-linux-musl/12.2.1/crtbeginS.o -L/riscv64-alpine-linux-musl/usr/lib/gcc/riscv64-alpine-linux-musl/12.2.1 -L/riscv64-alpine-linux-musl/usr/lib/gcc/riscv64-alpine-linux-musl/12.2.1/../../../../riscv64-alpine-linux-musl/lib -L/riscv64-alpine-linux-musl/lib -L/riscv64-alpine-linux-musl/usr/lib -rpath-link /riscv64-alpine-linux-musl/usr/lib -lssp_nonshared -lgcc --as-needed -lgcc_s --no-as-needed -lc -lgcc --as-needed -lgcc_s --no-as-needed /riscv64-alpine-linux-musl/usr/lib/gcc/riscv64-alpine-linux-musl/12.2.1/crtendS.o /riscv64-alpine-linux-musl/usr/lib/gcc/riscv64-alpine-linux-musl/12.2.1/crtn.o
#20 0.464 /usr/bin/riscv64-alpine-linux-musl-ld: -march=rv64i2p1_m2p0_a2p1_f2p2_d2p2_c2p0_zicsr2p0_zifencei2p0_zmmul1p0: unknown z ISA extension `zmmul'
#20 0.464 /usr/bin/riscv64-alpine-linux-musl-ld: failed to merge target specific data of file /riscv64-alpine-linux-musl/usr/lib/gcc/riscv64-alpine-linux-musl/12.2.1/crti.o
#20 0.464 ... rest of stderr output deleted ...
#20 0.464 configure:3198: $? = 1
#20 0.464 configure:3187: xx-clang -V >&5
#20 0.464 clang-15: error: argument to '-V' is missing (expected 1 value)
#20 0.464 clang-15: error: no input files
#20 0.464 configure:3198: $? = 1
#20 0.464 configure:3187: xx-clang -qversion >&5
#20 0.464 clang-15: error: unknown argument '-qversion'; did you mean '--version'?
#20 0.464 clang-15: error: no input files
#20 0.464 configure:3198: $? = 1
#20 0.464 configure:3218: checking whether the C compiler works
#20 0.464 configure:3240: xx-clang    conftest.c  >&5
#20 0.464 /usr/bin/riscv64-alpine-linux-musl-ld: -march=rv64i2p1_m2p0_a2p1_f2p2_d2p2_c2p0_zicsr2p0_zifencei2p0_zmmul1p0: unknown z ISA extension `zmmul'
#20 0.464 /usr/bin/riscv64-alpine-linux-musl-ld: failed to merge target specific data of file /riscv64-alpine-linux-musl/usr/lib/gcc/riscv64-alpine-linux-musl/12.2.1/crti.o
#20 0.464 /usr/bin/riscv64-alpine-linux-musl-ld: -march=rv64i2p1_m2p0_a2p1_f2p2_d2p2_c2p0_zicsr2p0_zifencei2p0_zmmul1p0: unknown z ISA extension `zmmul'
#20 0.464 /usr/bin/riscv64-alpine-linux-musl-ld: failed to merge target specific data of file /riscv64-alpine-linux-musl/usr/lib/gcc/riscv64-alpine-linux-musl/12.2.1/crtbeginS.o
#20 0.464 /usr/bin/riscv64-alpine-linux-musl-ld: failed to merge target specific data of file /tmp/conftest-72b27b.o
#20 0.464 /usr/bin/riscv64-alpine-linux-musl-ld: -march=rv64i2p1_m2p0_a2p1_f2p2_d2p2_c2p0_zicsr2p0_zifencei2p0_zmmul1p0: unknown z ISA extension `zmmul'
#20 0.464 /usr/bin/riscv64-alpine-linux-musl-ld: failed to merge target specific data of file /riscv64-alpine-linux-musl/usr/lib/gcc/riscv64-alpine-linux-musl/12.2.1/crtendS.o
#20 0.464 /usr/bin/riscv64-alpine-linux-musl-ld: -march=rv64i2p1_m2p0_a2p1_f2p2_d2p2_c2p0_zicsr2p0_zifencei2p0_zmmul1p0: unknown z ISA extension `zmmul'
#20 0.464 /usr/bin/riscv64-alpine-linux-musl-ld: failed to merge target specific data of file /riscv64-alpine-linux-musl/usr/lib/gcc/riscv64-alpine-linux-musl/12.2.1/crtn.o
#20 0.464 clang-15: error: linker command failed with exit code 1 (use -v to see invocation)
#20 0.464 configure:3244: $? = 1
#20 0.464 configure:3282: result: no
#20 0.464 configure: failed program was:
#20 0.464 | /* confdefs.h */
#20 0.464 | #define PACKAGE_NAME "libseccomp"
#20 0.464 | #define PACKAGE_TARNAME "libseccomp"
#20 0.464 | #define PACKAGE_VERSION "2.5.4"
#20 0.464 | #define PACKAGE_STRING "libseccomp 2.5.4"
#20 0.464 | #define PACKAGE_BUGREPORT ""
#20 0.464 | #define PACKAGE_URL ""
#20 0.464 | #define PACKAGE "libseccomp"
#20 0.464 | #define VERSION "2.5.4"
#20 0.464 | /* end confdefs.h.  */
#20 0.464 |
#20 0.464 | int
#20 0.464 | main ()
#20 0.464 | {
#20 0.464 |
#20 0.464 |   ;
#20 0.464 |   return 0;
#20 0.464 | }
#20 0.464 configure:3287: error: in `/usr/src/libseccomp':
#20 0.464 configure:3289: error: C compiler cannot create executables
#20 0.464 See `config.log' for more details
#20 0.464
#20 0.464 ## ---------------- ##
#20 0.464 ## Cache variables. ##
#20 0.464 ## ---------------- ##
#20 0.464
#20 0.464 ac_cv_env_CC_set='set'
#20 0.464 ac_cv_env_CC_value='xx-clang'
#20 0.464 ac_cv_env_CFLAGS_set=''
#20 0.464 ac_cv_env_CFLAGS_value=''
#20 0.464 ac_cv_env_CPPFLAGS_set=''
#20 0.464 ac_cv_env_CPPFLAGS_value=''
#20 0.464 ac_cv_env_CPP_set=''
#20 0.464 ac_cv_env_CPP_value=''
#20 0.464 ac_cv_env_LDFLAGS_set=''
#20 0.464 ac_cv_env_LDFLAGS_value=''
#20 0.464 ac_cv_env_LIBS_set=''
#20 0.464 ac_cv_env_LIBS_value=''
#20 0.464 ac_cv_env_LT_SYS_LIBRARY_PATH_set=''
#20 0.464 ac_cv_env_LT_SYS_LIBRARY_PATH_value=''
#20 0.464 ac_cv_env_PYTHON_set=''
#20 0.464 ac_cv_env_PYTHON_value=''
#20 0.464 ac_cv_env_build_alias_set=''
#20 0.464 ac_cv_env_build_alias_value=''
#20 0.464 ac_cv_env_host_alias_set='set'
#20 0.464 ac_cv_env_host_alias_value='riscv64-alpine-linux-musl'
#20 0.464 ac_cv_env_target_alias_set=''
#20 0.464 ac_cv_env_target_alias_value=''
#20 0.464 ac_cv_path_install='/usr/bin/install -c'
#20 0.464 ac_cv_prog_AWK='awk'
#20 0.464 ac_cv_prog_CC='xx-clang'
#20 0.464 ac_cv_prog_ac_ct_STRIP='strip'
#20 0.464 ac_cv_prog_make_make_set='yes'
#20 0.464 am_cv_make_support_nested_variables='yes'
#20 0.464 am_cv_prog_tar_pax='gnutar'
#20 0.464
#20 0.464 ## ----------------- ##
#20 0.464 ## Output variables. ##
#20 0.464 ## ----------------- ##
#20 0.464
#20 0.464 ACLOCAL='${SHELL} /usr/src/libseccomp/build-aux/missing aclocal-1.16'
#20 0.464 AMDEPBACKSLASH=''
#20 0.464 AMDEP_FALSE=''
#20 0.464 AMDEP_TRUE=''
#20 0.464 AMTAR='$${TAR-tar}'
#20 0.464 AM_BACKSLASH='\'
#20 0.464 AM_CFLAGS=''
#20 0.464 AM_CPPFLAGS=''
#20 0.464 AM_DEFAULT_V='$(AM_DEFAULT_VERBOSITY)'
#20 0.464 AM_DEFAULT_VERBOSITY='1'
#20 0.464 AM_LDFLAGS=''
#20 0.464 AM_V='$(V)'
#20 0.464 AR=''
#20 0.464 AUTOCONF='${SHELL} /usr/src/libseccomp/build-aux/missing autoconf'
#20 0.464 AUTOHEADER='${SHELL} /usr/src/libseccomp/build-aux/missing autoheader'
#20 0.464 AUTOMAKE='${SHELL} /usr/src/libseccomp/build-aux/missing automake-1.16'
#20 0.464 AWK='awk'
#20 0.464 CC='xx-clang'
#20 0.464 CCDEPMODE=''
#20 0.464 CFLAGS=''
#20 0.464 CODE_COVERAGE_CFLAGS=''
#20 0.464 CODE_COVERAGE_CPPFLAGS=''
#20 0.464 CODE_COVERAGE_CXXFLAGS=''
#20 0.464 CODE_COVERAGE_ENABLED=''
#20 0.464 CODE_COVERAGE_ENABLED_FALSE=''
#20 0.464 CODE_COVERAGE_ENABLED_TRUE=''
#20 0.464 CODE_COVERAGE_LDFLAGS=''
#20 0.464 CODE_COVERAGE_LIBS=''
#20 0.464 CODE_COVERAGE_RULES=''
#20 0.464 COVERITY_FALSE=''
#20 0.464 COVERITY_TRUE=''
#20 0.464 CPP=''
#20 0.464 CPPFLAGS=''
#20 0.464 CYGPATH_W='echo'
#20 0.464 DEFS=''
#20 0.464 DEPDIR=''
#20 0.464 DLLTOOL=''
#20 0.464 DSYMUTIL=''
#20 0.464 DUMPBIN=''
#20 0.464 ECHO_C=''
#20 0.464 ECHO_N='-n'
#20 0.464 ECHO_T=''
#20 0.464 EGREP=''
#20 0.464 ENABLE_PYTHON_FALSE=''
#20 0.464 ENABLE_PYTHON_TRUE=''
#20 0.464 EXEEXT=''
#20 0.464 FGREP=''
#20 0.464 GCOV=''
#20 0.464 GENHTML=''
#20 0.464 GPERF=''
#20 0.464 GREP=''
#20 0.464 INSTALL_DATA='${INSTALL} -m 644'
#20 0.464 INSTALL_PROGRAM='${INSTALL}'
#20 0.464 INSTALL_SCRIPT='${INSTALL}'
#20 0.464 INSTALL_STRIP_PROGRAM='$(install_sh) -c -s'
#20 0.464 LCOV=''
#20 0.464 LD=''
#20 0.464 LDFLAGS=''
#20 0.464 LIBOBJS=''
#20 0.464 LIBS=''
#20 0.464 LIBTOOL=''
#20 0.464 LIPO=''
#20 0.464 LN_S=''
#20 0.464 LTLIBOBJS=''
#20 0.464 LT_SYS_LIBRARY_PATH=''
#20 0.464 MAKEINFO='${SHELL} /usr/src/libseccomp/build-aux/missing makeinfo'
#20 0.464 MANIFEST_TOOL=''
#20 0.464 MKDIR_P='build-aux/install-sh -c -d'
#20 0.464 NM=''
#20 0.464 NMEDIT=''
#20 0.464 OBJDUMP=''
#20 0.464 OBJEXT=''
#20 0.464 OTOOL64=''
#20 0.464 OTOOL=''
#20 0.464 PACKAGE='libseccomp'
#20 0.464 PACKAGE_BUGREPORT=''
#20 0.464 PACKAGE_NAME='libseccomp'
#20 0.464 PACKAGE_STRING='libseccomp 2.5.4'
#20 0.464 PACKAGE_TARNAME='libseccomp'
#20 0.464 PACKAGE_URL=''
#20 0.464 PACKAGE_VERSION='2.5.4'
#20 0.464 PATH_SEPARATOR=':'
#20 0.464 PYTHON=''
#20 0.464 PYTHON_EXEC_PREFIX=''
#20 0.464 PYTHON_PLATFORM=''
#20 0.464 PYTHON_PREFIX=''
#20 0.464 PYTHON_VERSION=''
#20 0.464 RANLIB=''
#20 0.464 SED=''
#20 0.464 SET_MAKE=''
#20 0.464 SHELL='/bin/sh'
#20 0.464 STRIP='strip'
#20 0.464 VERSION='2.5.4'
#20 0.464 VERSION_MAJOR=''
#20 0.464 VERSION_MICRO=''
#20 0.464 VERSION_MINOR=''
#20 0.464 ac_ct_AR=''
#20 0.464 ac_ct_CC=''
#20 0.464 ac_ct_DUMPBIN=''
#20 0.464 am__EXEEXT_FALSE=''
#20 0.464 am__EXEEXT_TRUE=''
#20 0.464 am__fastdepCC_FALSE=''
#20 0.464 am__fastdepCC_TRUE=''
#20 0.464 am__include=''
#20 0.464 am__isrc=''
#20 0.464 am__leading_dot='.'
#20 0.464 am__nodep=''
#20 0.464 am__quote=''
#20 0.464 am__tar='tar --format=posix -chf - "$$tardir"'
#20 0.464 am__untar='tar -xf -'
#20 0.464 bindir='${exec_prefix}/bin'
#20 0.464 build=''
#20 0.464 build_alias=''
#20 0.464 build_cpu=''
#20 0.464 build_os=''
#20 0.464 build_vendor=''
#20 0.464 cython=''
#20 0.464 datadir='${datarootdir}'
#20 0.464 datarootdir='${prefix}/share'
#20 0.464 docdir='${datarootdir}/doc/${PACKAGE_TARNAME}'
#20 0.464 dvidir='${docdir}'
#20 0.464 exec_prefix='NONE'
#20 0.464 have_coverity=''
#20 0.464 host='riscv64-alpine-linux-musl'
#20 0.464 host_alias='riscv64-alpine-linux-musl'
#20 0.464 host_cpu=''
#20 0.464 host_os=''
#20 0.464 host_vendor=''
#20 0.464 htmldir='${docdir}'
#20 0.464 includedir='${prefix}/include'
#20 0.464 infodir='${datarootdir}/info'
#20 0.464 install_sh='${SHELL} /usr/src/libseccomp/build-aux/install-sh'
#20 0.464 libdir='${exec_prefix}/lib'
#20 0.464 libexecdir='${exec_prefix}/libexec'
#20 0.464 localedir='${datarootdir}/locale'
#20 0.464 localstatedir='${prefix}/var'
#20 0.464 mandir='${datarootdir}/man'
#20 0.464 mkdir_p='$(MKDIR_P)'
#20 0.464 oldincludedir='/usr/include'
#20 0.464 pdfdir='${docdir}'
#20 0.464 pkgpyexecdir=''
#20 0.464 pkgpythondir=''
#20 0.464 prefix='NONE'
#20 0.464 program_transform_name='s,x,x,'
#20 0.464 psdir='${docdir}'
#20 0.464 pyexecdir=''
#20 0.464 pythondir=''
#20 0.464 runstatedir='${localstatedir}/run'
#20 0.464 sbindir='${exec_prefix}/sbin'
#20 0.464 sharedstatedir='${prefix}/com'
#20 0.464 sysconfdir='${prefix}/etc'
#20 0.464 target_alias=''
#20 0.464 
#20 0.464 ## ----------- ##
#20 0.464 ## confdefs.h. ##
#20 0.464 ## ----------- ##
#20 0.464
#20 0.464 /* confdefs.h */
#20 0.464 #define PACKAGE_NAME "libseccomp"
#20 0.464 #define PACKAGE_TARNAME "libseccomp"
#20 0.464 #define PACKAGE_VERSION "2.5.4"
#20 0.464 #define PACKAGE_STRING "libseccomp 2.5.4"
#20 0.464 #define PACKAGE_BUGREPORT ""
#20 0.464 #define PACKAGE_URL ""
#20 0.464 #define PACKAGE "libseccomp"
#20 0.464 #define VERSION "2.5.4"
#20 0.464
#20 0.464 configure: exit 77
#20 0.464 + make install
#20 0.466 make: *** No rule to make target 'install'.  Stop.
```

cc @tonistiigi @AkihiroSuda

--- a/Dockerfile
+++ b/Dockerfile
@@ -32,7 +32,7 @@ FROM alpine:edge@sha256:c223f84e05c23c0571ce8decefef818864869187e1a3ea47719412e2
 FROM alpine-$TARGETARCH AS alpinebase
 
 # xx is a helper for cross-compilation
-FROM --platform=$BUILDPLATFORM tonistiigi/xx:1.1.2 AS xx
+FROM --platform=$BUILDPLATFORM tonistiigi/xx:1.2.1 AS xx
 
 # go base image
 FROM --platform=$BUILDPLATFORM golang:1.19-alpine${ALPINE_VERSION} AS golatest

